services/backend/src/recommendation_system/recommendation_flow/model_prediction/AlphaModel.py

```diff
@@ -89,10 +82,17 @@ def predict_probabilities(self, content_ids, user_id, seed=None, **kwargs):
                 )
             )
             
+<<<<<<< HEAD
```

Comment:
you need to remove these

Reply:
Thank you for your recommendation. Now I have removed the commented lines and used the index / content_id dictionary to get the id_to_embedding directly from the PKL file. Do you mind taking a look at the new ones?